benefits/eligibility/views.py

```diff
@@ -58,12 +58,12 @@ def index(request):
 def start(request):
     """View handler for the eligibility verification getting started screen."""
 
-    session.update(request, eligibility_types=[])
+    session.update(request, eligibility_types=[], origin=reverse("eligibility:start"))
     verifier = session.verifier(request)
 
     if verifier.requires_authentication:
         auth_provider = verifier.auth_provider
-        button = viewmodels.Button.external(text=_(auth_provider.sign_in_button_label), url="#", id="login")
+        button = viewmodels.Button.external(text=_(auth_provider.sign_in_button_label), url=reverse("oauth:login"), id="login")
```

Comment:
😕 I am getting a `NoReverseMatch` error from this `reverse("oauth:login")`. Do you have any ideas on what might be going on?

![image](https://user-images.githubusercontent.com/25497886/163032881-b6bbb4a3-3e26-4e6c-a5c1-34ba67f91eb5.png)

Reply:
So this is probably related to [@afeld comment above](https://github.com/cal-itp/benefits/pull/414#issuecomment-1097083063), in that the [`.env` needs to be updated with the new keys/values](https://github.com/cal-itp/benefits/pull/414/files#diff-e43af9f217e778e159273c8b2c475150122ad0ccf0525cb2bce1edd237ca5f03) and reread.

We could put a check here to ensure that the `OAUTH_CLIENT_NAME` setting has a value or raise an Exception, which would give a clearer error message than this one.

Reply:
> We could put a check here to ensure that the `OAUTH_CLIENT_NAME` setting has a value or raise an Exception, which would give a clearer error message than this one.

I think this is a good idea. I vote we do this ☑️